tools/release-stable.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+import del from 'del';
+import fs from 'fs-extra';
+import git from 'simple-git/promise';
+import path from 'path';
+
+const repoURL = `https://${process.env.GH_TOKEN}@github.com/grommet/grommet-theme-hpe.git`;
+const localFolder = path.resolve('.tmp/grommet-theme-hpe');
+const localDist = path.resolve('dist');
+
+if (process.env.CI) {
```

Comment:
Small nit. I believe all environment variables evaluate as strings, so this block will always execute regardless of the value assigned to `CI` (e.g. if CI=false, this block will still execute). Suggestion: `if (process.env.CI === 'true')`